mlrun/utils/helpers.py

```diff
@@ -47,6 +47,10 @@
 hub_prefix = "hub://"
 DB_SCHEMA = "store"
 
+legal_time_units = ["year", "month", "day", "hour", "minute", "second"]
```

Comment:
```suggestion
legal_time_partition_units = ["year", "month", "day", "hour", "minute", "second"]
```

Not critical, but it's just for partitioning, so not to confuse that these are all the legal units we know of (though it's pretty much the same...)

Reply:
I'm a bit assitant to change it because
1. It could be used for other purposes and
2. the same constant is defined in storey, and I'm worried that the inconsistency will be confusing (or of course we could change it there too...)